### PR TITLE
Remove qiskit.org links

### PR DESCRIPTION
**Pull request by Eric-Arellano**
qiskit.org has been removed.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -49,9 +49,6 @@ please ensure that:
    If your pull request is adding a new class, function, or module that is
    intended to be user facing ensure that you've also added those to a
    documentation `autosummary` index to include it in the api documentation.
-   For more details you can refer to:
```
Comment (Eric-Arellano) [inline CONTRIBUTING.md:52]: Doesn't exist anymore.

**Comment by Eric-Arellano** [comment]
Gentle bump @grace-harper

**Review by quantumjim — APPROVED** [review]

**Comment by Eric-Arellano** [comment]
Thanks! I don't have write access so this will need to have someone hit the merge button.

**Review by grace-harper — APPROVED** [review]
LGTM!

**Pull request merged**